libshaderc/src/shaderc_test.cc

```diff
@@ -111,327 +114,299 @@ class Compiler {
   shaderc_compiler_t compiler;
 };
 
-// Compiles a shader and returns true on success, false on failure.
-bool CompilationSuccess(const shaderc_compiler_t compiler,
-                        const std::string& shader, shaderc_shader_kind kind) {
-  return shaderc_module_get_success(
-      Compilation(compiler, shader, kind).result());
-}
+// A testing class to test the compilation of string with or without options.
```

Comment:
Good blurb.
Minor grammar problem:  should be  "test the compilation of a string".  (Was missing "a")